src/xml.c

```diff
@@ -281,25 +288,35 @@ gchar *
 xhtml_strip_dhtml (const gchar *html)
 {
 	if (!dhtml_strippers) {
-		xhtml_stripper_add (&dhtml_strippers, "\\s+onload='[^']+'");
-		xhtml_stripper_add (&dhtml_strippers, "\\s+onload=\"[^\"]+\"");
-		xhtml_stripper_add (&dhtml_strippers, "<\\s*script\\s*>.*</\\s*script\\s*>");
-		xhtml_stripper_add (&dhtml_strippers, "<\\s*meta\\s*>.*</\\s*meta\\s*>");
-		xhtml_stripper_add (&dhtml_strippers, "<\\s*iframe[^>]*\\s*>.*</\\s*iframe\\s*>");
+		xhtml_regex_add (&dhtml_strippers, "\\s+onload='[^']+'", g_strdup(""));
+		xhtml_regex_add (&dhtml_strippers, "\\s+onload=\"[^\"]+\"", g_strdup(""));
+		// xhtml_regex_add (&dhtml_strippers, "<\\s*script\\s*>.*</\\s*script\\s*>", g_strdup(""));
```

Comment:
I commented out the lines rather than deleted because they didn't work prior to this fix.
If they had worked, embedded videos would not have worked before as this code would have stripped them.

Reply:
I wondered about the requirement for stripping scripts and iframes and it's intent, especially since has not worked since it was added.

Reply:
It not being working is my fault for so long not implementing proper tests.

I'll do this as a followup. Stripping the iframes was very important in the past as many ads where iframe embedded in the feed content. Nowadays it less relevant for the feed content. But I still consider it as a bug that it is not removed. I'll fix this once we have failing tests for this :-)

About videos they are not really a problem for several reasons
- in Reader Mode they are stripped anyway
- HTML5 embedded videos are not affected
- we have XSLT base re-embedding of youtube videos